From c4398eb44285405a919bd3dfeb63d7faf7cbaf21 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Tue, 16 May 2017 14:39:49 +0100
Subject: [PATCH 128/215] mmc: Change downstream MMC driver CONFIG option

The upstream SDHOST driver has now claimed CONFIG_MMC_BCM2835, which
clashes with the downstream MMC driver. Rename the downstream option to
CONFIG_MMC_BCM2835_MMC.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 drivers/mmc/host/Kconfig  | 4 ++--
 drivers/mmc/host/Makefile | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mmc/host/Kconfig b/drivers/mmc/host/Kconfig
index 6300af1..e577d20 100644
--- a/drivers/mmc/host/Kconfig
+++ b/drivers/mmc/host/Kconfig
@@ -4,7 +4,7 @@
 
 comment "MMC/SD/SDIO Host Controller Drivers"
 
-config MMC_BCM2835
+config MMC_BCM2835_MMC
 	tristate "MMC support on BCM2835"
 	depends on MACH_BCM2708 || MACH_BCM2709 || ARCH_BCM2835
 	help
@@ -16,7 +16,7 @@ config MMC_BCM2835
 
 config MMC_BCM2835_DMA
 	bool "DMA support on BCM2835 Arasan controller"
-	depends on MMC_BCM2835
+	depends on MMC_BCM2835_MMC
 	help
 	  Enable DMA support on the Arasan SDHCI controller in Broadcom 2708
 	  based chips.
diff --git a/drivers/mmc/host/Makefile b/drivers/mmc/host/Makefile
index f4b8951..d352fab 100644
--- a/drivers/mmc/host/Makefile
+++ b/drivers/mmc/host/Makefile
@@ -18,7 +18,7 @@ obj-$(CONFIG_MMC_SDHCI_S3C)	+= sdhci-s3c.o
 obj-$(CONFIG_MMC_SDHCI_SIRF)   	+= sdhci-sirf.o
 obj-$(CONFIG_MMC_SDHCI_F_SDH30)	+= sdhci_f_sdh30.o
 obj-$(CONFIG_MMC_SDHCI_SPEAR)	+= sdhci-spear.o
-obj-$(CONFIG_MMC_BCM2835)	+= bcm2835-mmc.o
+obj-$(CONFIG_MMC_BCM2835_MMC)	+= bcm2835-mmc.o
 obj-$(CONFIG_MMC_BCM2835_SDHOST)	+= bcm2835-sdhost.o
 obj-$(CONFIG_MMC_WBSD)		+= wbsd.o
 obj-$(CONFIG_MMC_AU1X)		+= au1xmmc.o
-- 
1.9.1

